app/src/main/java/com/hedvig/app/feature/crossselling/ui/CrossSellingResultScreen.kt

```diff
@@ -0,0 +1,180 @@
+package com.hedvig.app.feature.crossselling.ui
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.compose.ui.unit.dp
+import com.hedvig.app.R
+import com.hedvig.app.ui.compose.HedvigTheme
+import com.hedvig.app.ui.compose.designsystem.LargeContainedButton
+import com.hedvig.app.ui.compose.designsystem.LargeOutlinedButton
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+// TODO text resources still missing
```

Comment:
Do we wait for string resources to exist before merge?

Reply:
Yes 🙂